tests/wpt/web-platform-tests/webmessaging/Channel_postMessage_with_transfer_outoing_messages.htm

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html>
+<head>
+<title> When transferring a port,
+    outgoing messages sent at each transfer step are received in order by the entangled port. </title>
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+</head>
+<body>
+<div id=log></div>
+<script>
+async_test(function(t) {
```

Comment:
Because of supporting `postMessage` and `MessageChannel` in both `Window` and `Worker`, I'm wondering maybe we can have the tests added in this PR in `.any.js` format 👀?

Reply:
Good point, yes they could be re-written in that style.